### YamlMime:UniversalReference
api_name: []
items:
- children:
  - azure.mgmt.sql.operations.FailoverGroupsOperations.create_or_update
  - azure.mgmt.sql.operations.FailoverGroupsOperations.delete
  - azure.mgmt.sql.operations.FailoverGroupsOperations.failover
  - azure.mgmt.sql.operations.FailoverGroupsOperations.force_failover_allow_data_loss
  - azure.mgmt.sql.operations.FailoverGroupsOperations.get
  - azure.mgmt.sql.operations.FailoverGroupsOperations.list_by_server
  - azure.mgmt.sql.operations.FailoverGroupsOperations.update
  class: azure.mgmt.sql.operations.FailoverGroupsOperations
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations
  inheritance:
  - type: builtins.object
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: FailoverGroupsOperations
  source:
    id: FailoverGroupsOperations
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 20
  summary: 'FailoverGroupsOperations operations.








    '
  syntax:
    content: FailoverGroupsOperations(client, config, serializer, deserializer)
    parameters:
    - description: 'Client for service requests.

        '
      id: client
    - description: 'Configuration of service client.

        '
      id: config
    - description: 'An object model serializer.

        '
      id: serializer
    - description: 'An objec model deserializer.

        '
      id: deserializer
    variables:
    - description: 'The API version to use for the request. Constant value: "2015-05-01-preview".

        '
      id: api_version
  type: class
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.create_or_update
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: create_or_update
  source:
    id: create_or_update
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 106
  summary: 'Creates or updates a failover group.

    '
  syntax:
    content: create_or_update(resource_group_name, server_name, failover_group_name,
      parameters, custom_headers=None, raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - description: 'The name of the failover group.

        '
      id: failover_group_name
      type:
      - str
    - description: 'The failover group parameters.

        '
      id: parameters
      type:
      - azure.mgmt.sql.models.FailoverGroup
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    return:
      type:
      - 'msrestazure.azure_operation.AzureOperationPoller

        instance that returns @azure.mgmt.sql.models.FailoverGroup'
      - 'msrest.pipeline.ClientRawResponse

        if raw=true'
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.create_or_update
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.delete
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: delete
  source:
    id: delete
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 206
  summary: 'Deletes a failover group.

    '
  syntax:
    content: delete(resource_group_name, server_name, failover_group_name, custom_headers=None,
      raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - description: 'The name of the failover group.

        '
      id: failover_group_name
      type:
      - str
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    return:
      type:
      - 'msrestazure.azure_operation.AzureOperationPoller

        instance that returns None'
      - 'msrest.pipeline.ClientRawResponse

        if raw=true'
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.delete
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.failover
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: failover
  source:
    id: failover
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 459
  summary: 'Fails over from the current primary server to this server.

    '
  syntax:
    content: failover(resource_group_name, server_name, failover_group_name, custom_headers=None,
      raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - description: 'The name of the failover group.

        '
      id: failover_group_name
      type:
      - str
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    return:
      type:
      - 'msrestazure.azure_operation.AzureOperationPoller

        instance that returns @azure.mgmt.sql.models.FailoverGroup'
      - 'msrest.pipeline.ClientRawResponse

        if raw=true'
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.failover
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.force_failover_allow_data_loss
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: force_failover_allow_data_loss
  source:
    id: force_failover_allow_data_loss
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 550
  summary: 'Fails over from the current primary server to this server. This

    operation might result in data loss.

    '
  syntax:
    content: force_failover_allow_data_loss(resource_group_name, server_name, failover_group_name,
      custom_headers=None, raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - description: 'The name of the failover group.

        '
      id: failover_group_name
      type:
      - str
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    return:
      type:
      - 'msrestazure.azure_operation.AzureOperationPoller

        instance that returns @azure.mgmt.sql.models.FailoverGroup'
      - 'msrest.pipeline.ClientRawResponse

        if raw=true'
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.force_failover_allow_data_loss
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.get
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: get
  source:
    id: get
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 39
  summary: 'Gets a failover group.

    '
  syntax:
    content: get(resource_group_name, server_name, failover_group_name, custom_headers=None,
      raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - description: 'The name of the failover group.

        '
      id: failover_group_name
      type:
      - str
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    - description: '*Operation configuration

        overrides*.

        '
      id: operation_config
    return:
      type:
      - azure.mgmt.sql.models.FailoverGroup
      - 'msrest.pipeline.ClientRawResponse

        if raw=true'
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.get
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.list_by_server
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: list_by_server
  source:
    id: list_by_server
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 387
  summary: 'Lists the failover groups in a server.

    '
  syntax:
    content: list_by_server(resource_group_name, server_name, custom_headers=None,
      raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    - description: '*Operation configuration

        overrides*.

        '
      id: operation_config
    return:
      type:
      - azure.mgmt.sql.models.FailoverGroupPaged
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.list_by_server
- class: azure.mgmt.sql.operations.FailoverGroupsOperations
  exceptions:
  - type: msrestazure.azure_exceptions.CloudError
  fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.update
  langs:
  - python
  module: azure.mgmt.sql.operations
  name: update
  source:
    id: update
    path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
    remote:
      branch: master
      path: azure-mgmt-sql\azure\mgmt\sql\operations\failover_groups_operations.py
      repo: https://github.com/Azure/azure-sdk-for-python.git
    startLine: 289
  summary: 'Updates a failover group.

    '
  syntax:
    content: update(resource_group_name, server_name, failover_group_name, parameters,
      custom_headers=None, raw=False, **operation_config)
    parameters:
    - description: 'The name of the resource group that

        contains the resource. You can obtain this value from the Azure

        Resource Manager API or the portal.

        '
      id: resource_group_name
      type:
      - str
    - description: 'The name of the server containing the failover

        group.

        '
      id: server_name
      type:
      - str
    - description: 'The name of the failover group.

        '
      id: failover_group_name
      type:
      - str
    - description: 'The failover group parameters.

        '
      id: parameters
      type:
      - azure.mgmt.sql.models.FailoverGroup
    - defaultValue: None
      description: 'headers that will be added to the request

        '
      id: custom_headers
      type:
      - dict
    - defaultValue: 'False'
      description: 'returns the direct response alongside the

        deserialized response

        '
      id: raw
      type:
      - bool
    return:
      type:
      - 'msrestazure.azure_operation.AzureOperationPoller

        instance that returns @azure.mgmt.sql.models.FailoverGroup'
      - 'msrest.pipeline.ClientRawResponse

        if raw=true'
  type: method
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.update
references:
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.create_or_update
  isExternal: false
  name: create_or_update
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.create_or_update
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.delete
  isExternal: false
  name: delete
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.delete
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.failover
  isExternal: false
  name: failover
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.failover
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.force_failover_allow_data_loss
  isExternal: false
  name: force_failover_allow_data_loss
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.force_failover_allow_data_loss
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.get
  isExternal: false
  name: get
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.get
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.list_by_server
  isExternal: false
  name: list_by_server
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.list_by_server
- fullName: azure.mgmt.sql.operations.FailoverGroupsOperations.update
  isExternal: false
  name: update
  parent: azure.mgmt.sql.operations.FailoverGroupsOperations
  uid: azure.mgmt.sql.operations.FailoverGroupsOperations.update
